commit/core.pyx

```diff
@@ -141,7 +141,7 @@ cdef class Evaluation :
         print( '\t* DWI signal:' )
         self.set_config('dwi_filename', dwi_filename)
         self.niiDWI  = nibabel.load( pjoin( self.get_config('DATA_path'), dwi_filename) )
-        self.niiDWI_img = self.niiDWI.get_fdata().astype(np.float32)
+        self.niiDWI_img = np.asanyarray( self.niiDWI.dataobj ).astype(np.float32)
```

Comment:
Why is `get_fdata()` not ok?

Reply:
It can use much memory and could create some problems ([nibabel get_data vs get_fdata](https://github.com/nipy/dmriprep/issues/55)).